docs/developers_guide/copyright_license.md

```diff
@@ -1,46 +1,206 @@
+
+__Can I copyright code to a generic "SuperDARN" organization?__<br/>
+No, the copyright holder has to be a legal entity or a person. Your employer may also object to this. If you contribute to the RST in your free time, then just put your own name in the copyright line.
+
+__Why does every RST source file have license notices at the top? Isn't it sufficient to include the license file in the top-level directory of RST?__<br/>
+Since the GPL allows users to modify and redistribute portions of the RST, it is possible that individual source files might become separated from the license file. If this happens, it will be unclear to users what their legal rights are to use/modify/distribute that version of the code (which is a violation of the GPL). Therefore, all source files should clearly indicate that they are licensed under the GPL (writing "see license.txt" is not sufficient).
+
+
+__If I add a new library to the RST, can that library have a different license?__<br/>
+No, the GPL requires that the whole "work" be released with the same license.
+
+
+
+__What are the licensing consequences of copying code from the RST into my own software project?__
+
+- You are free to copy or modify any portion of the RST code
+- You are free to share the software outputs (data files, plots)
+- If you choose to share the binaries for your new software: 
+    - You must also share the source code
+    - Your software must be licensed under GPL (v3 or later)
+
+
+__I've added code to the RST. Can I also release my code under a different license?__<br/>
+Yes, provided that you are the copyright holder of the code, you are free to license it under different non-exclusive licenses ([more info](https://www.gnu.org/licenses/gpl-faq.html#ReleaseUnderGPLAndNF)). Remember that:
+
+- Once your code is added to the RST, that version of the code is licensed under GPLv3, and you cannot revoke this
+- If your software includes other code from the RST (or other GPL-licensed code), your software can only be licensed under GPL. 
+
+
+__Who owns the outputs of RST (processed data files, plots)?__<br/>
+RST outputs belong to the end user who created them. The user is free to share them under any terms, and is not bound by the GPL.
+
+__I have more questions about the GPL__<br/>
+There's lots of helpful information [here](https://www.gnu.org/licenses/gpl-faq.html).
+
+
+
+
+## History of RST licensing
+
+RST was originally developed at the Johns Hopkins University Applied Physics Laboratory (JHUAPL). Around 2012, JHUAPL granted permission for RST to be re-licensed so that the SuperDARN community could continue maintaining the software collaboratively. This process has caused confusion over the years, since even scientists who are expert programmers may not be familiar with the intricacies of software licensing and copyright.
```

Comment:
``` suggestion
RST was originally developed at the Johns Hopkins University/Applied Physics Laboratory (JHU/APL). Around 2012, JHU/APL granted permission for RST to be re-licensed so that the SuperDARN community could continue maintaining the software collaboratively. This process has caused confusion over the years, since even scientists who are expert programmers may not be familiar with the intricacies of software licensing and copyright.
```